device/common/include/traccc/seeding/device/select_seeds.hpp

```diff
@@ -1,3 +1,10 @@
+/** TRACCC library, part of the ACTS project (R&D line)
+ *
+ * (c) 2021-2022 CERN for the benefit of the ACTS project
+ *
+ * Mozilla Public License Version 2.0
+ */
+
```

Comment:
Thank you for this correction. Is it worth adding a unit test for this?

Reply:
I don't think we managed to figure out how to efficiently test for this in the ATLAS offline code either. :frowning: I wouldn't spend too much time on this, unless you already know how the CI could be taught to test for this.